src/backend/columnar/columnar_reader.c

```diff
@@ -560,17 +561,41 @@ FreeChunkData(ChunkData *chunkData)
 }
 
 
-/* ColumnarTableRowCount returns the exact row count of a table using skiplists */
+/*
+ * ColumnarTableRowCount returns the row count of a table using stripe skiplists.
+ */
 uint64
 ColumnarTableRowCount(Relation relation)
 {
-	ListCell *stripeMetadataCell = NULL;
-	uint64 totalRowCount = 0;
 	List *stripeList = StripesForRelfilenode(relation->rd_node);
+	return StripeMetadataListRowCount(stripeList);
+}
+
 
-	foreach(stripeMetadataCell, stripeList)
+/*
+ * ColumnarTableRowCount returns the row count of the storage with storageId
+ * using stripe skiplists.
+ */
+uint64
+ColumnarStorageRowCount(uint64 storageId)
```

Comment:
Let's try to keep a consistent naming convention. ColumnarStorage* right now refers to functions defined in columnar_storage.c. Also, this function is not really looking at the storage at all, it's just using the storageId, which is mostly a metadata concept (columnar_storage.c doesn't care much about storage ID, really, it's just a handle).

Reply:
now I reverted that commit, so we don't have such a function anymore